pkgs/build-support/writers/default.nix

```diff
@@ -245,7 +245,7 @@ rec {
   } /* sh */ ''
     # nginx-config-formatter has an error - https://github.com/1connect/nginx-config-formatter/issues/16
     awk -f ${awkFormatNginx} "$textPath" | sed '/^\s*$/d' > $out
-    gixy $out
+    gixy --skips host_spoofing $out
```

Comment:
You typically want this check though, there's just one case on the deployment manager configuration where we had no choice but to spoof the host so that the AWS signing verification algorithm in `minio` wouldn't throw a fit.